crates/trie/sparse-parallel/src/trie.rs

```diff
@@ -18,6 +18,13 @@ use reth_trie_sparse::{
 use smallvec::SmallVec;
 use tracing::trace;
 
+/// The maximum length of a path, in nibbles, which belongs to the upper subtrie of a
+/// [`ParallelSparseTrie`]. All longer paths belong to a lower subtrie.
+const UPPER_TRIE_MAX_DEPTH: usize = 2;
+
+/// Number of lower subtries which are managed by the [`ParallelSparseTrie`].
+const NUM_LOWER_SUBTRIES: usize = 1 << (UPPER_TRIE_MAX_DEPTH * 4);
```

Comment:
can we do `pow` here for readability? 😅

Reply:
Done, didn't realize it was a `const fn` :laughing: